databroker/tests/test_broker.py

```diff
@@ -389,6 +389,11 @@ def test_filters(db_empty, RE):
 
     assert header['start']['uid'] == ken_calib_uid
 
+    db.clear_filters()
+    db.add_filter(start_time='2017')
+    db.add_filter(start_time='2017')
```

Comment:
then we will have two start_time as 2017 and 2018. I can add more tests to double check.